src/devices/cpu/i386/i386op32.hxx

```diff
@@ -318,7 +318,7 @@ void i386_device::i386_bsr_r32_rm32()      // Opcode 0x0f bd
 	} else {
 		m_ZF = 0;
 		dst = temp = 31;
-		while( (src & (1 << temp)) == 0 ) {
+		while( (src & (1u << temp)) == 0 ) {
```

Comment:
Please use uppercase `U` and `L` suffixes for literals - it's easy to visually confuse lowercase `l` with `1`, so we encourage uppercase for both.